modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -829,6 +834,34 @@ class NDB_Form_User_Accounts extends NDB_Form
     }
 
     /**
+    * Query database on UserID to check that password has doesn't
+    * exist and account is still pending approval, to ensure the
+    * users with account activity can't be rejected by setting
+    * their status to pending.
+    *
+    * @param string $id identifier of account to verify
+    *
+    * @return boolean true is Password_hash is null and
+    * Pending_approval is Yes, false otherwsie
+    */
+    function _canRejectAccount($id)
```

Comment:
@driusan so I should redeclare this function as:  ``` static function canRejectAccount($id) ``` ?
How can I then call this function from within my ajax script? Do I need to instantiate an NDB_Form_user_accounts object inside my ajax script?